mast_aladin_lite/app.py

```diff
@@ -1,20 +1,65 @@
+from ipywidgets import widgets
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from astropy.utils.masked import Masked
+
 from ipyaladin import Aladin
 from mast_aladin_lite.aida import AID
+from mast_aladin_lite.table import MastTable
 
 
 # store reference to the latest instantiation:
 _latest_instantiated_app = None
 
 
 class MastAladin(Aladin):
+    table_widgets = dict()
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+        # the `aid` attribute gives access to methods from the
+        # Astro Image Display (AID) API
         self.aid = AID(self)
 
+        self.selected_table = widgets.Output()
+
         global _latest_instantiated_app
         _latest_instantiated_app = self
 
+    def load_table(self, table, name='table', load_footprints=True, update_viewport=True):
+        table_widget = MastTable(table)
+        self.table_widgets[name] = self.selected_table = table_widget
+
+        if load_footprints:
+            if 's_region' in table.colnames:
+                self.add_graphic_overlay_from_stcs(table['s_region'])
+            else:
+                raise ValueError(
+                    "The table does not contain an `s_region` column, so no "
+                    "footprints can be loaded."
+                )
+
+        if update_viewport:
+            ra_column, dec_column = 'ra', 'dec'
+
+            if ra_column not in table.colnames:
+                ra_column, dec_column = 'targ_ra', 'targ_dec'
```

Comment:
comment: this seems like its got a potential to error out if neither `ra` or `targ_ra` is in the table. We might want to consider a check like this 
```python
if 'ra' in table.colnames:
    ra_column, dec_column = 'ra', 'dec'
elif 'targ_ra' in table.colnames:
    ra_column, dec_column = 'targ_ra', 'targ_dec'
else:
    raise ValueError(
"""
No valid value for 'ra' and 'dec' column found, 
unable to update the viewport. Accepted values 
are 'ra'/'dec' and 'targ_ra'/'targ_dec'.
""")
```